Dockerfile

```diff
@@ -89,6 +89,7 @@ ADD runtimes/python3-9 /workspace
 RUN chown -R 9653 /.cache /.local /workspace
 WORKDIR /workspace
 USER 9653:9653
+RUN pip3 install git+https://github.com/argoproj-labs/argo-dataflow#subdirectory=sdks/python
```

Comment:
Should this go into requirements.txt?

Reply:
It should but I'm not sure where this requirements.txt lives. I couldn't find files that go into this image...